.github/workflows/email_checker.yml

```
@@ -0,0 +1,10 @@
+name: PR email addresses checker
+
+on: pull_request
```

Since we only expect PRs on `qcom-next-staging` (see https://github.com/qualcomm-linux/kernel/pull/3#issuecomment-2896661554), configure this accordingly.
```suggestion
on:
  pull_request:
    branches: [ qcom-next-staging ]
```